docs/en/operations/utilities/clickhouse-local.md

```diff
@@ -6,25 +6,26 @@ sidebar_label: clickhouse-local
 
 # clickhouse-local 
 
-The `clickhouse-local` program enables you to perform fast processing on local files, without having to deploy and configure the ClickHouse server.
+The `clickhouse-local` program enables you to perform fast processing on local files, without having to deploy and configure the ClickHouse server. It accepts data that represent tables and queries them using [ClickHouse SQL dialect](../../sql-reference/). `clickhouse-local` uses the same core as ClickHouse server, so it supports most of the features and the same set of formats and table engines. 
 
-Accepts data that represent tables and queries them using [ClickHouse SQL dialect](../../sql-reference/).
+By default `clickhouse-local` does not have access to data on the same host, but it supports loading server configuration using `--config-file` argument. For temporary data, a unique temporary data directory is created by default.
```

Comment:
This is slightly incorrect - in fact, clickhouse-local can process any files that the user has access to. But it will not attach the data from clickhouse-server on the same machine.

Reply:
I didn't really change the previous verbiage, that is what was there :)